main.go

```diff
@@ -613,18 +612,16 @@ func main() {
 	}
 
 	// exporting xcresult only if test result dir is present
-	if testResultPath := os.Getenv(bitriseConfigs.BitriseTestResultDirEnvKey); len(testResultPath) > 0 {
```

Comment:
pls do a dep update, the env key is renamed and may the value also changed: `BitrisePerStepTestResultDirEnvKey`